front/app/component-library/components/CollapsibleContainer/index.tsx

```diff
@@ -46,9 +33,9 @@ const TitleButton = styled.button`
     }
   }
 
-  &:focus-visible {
```

Comment:
This didn't work consitently. Button lost focus styles after collapsing the container.

Reply:
@brentguf Hmm I don't see that on my end - what browser are you using?

[Screencast from 2024-11-13 15:25:50.webm](https://github.com/user-attachments/assets/36233c9d-42fe-434c-af6a-e32b3f32e9b7)

Reply:
@brentguf Also I feel like we shouldn't show this focus border when you've clicked on the element with the mouse (that's not a convention we do for other UI elements either I think - we just show the focus border for keyboard navigation).

By using the `focus` instead of `focus-visible`, I don't think the UI looks super nice. I think the fact that the content is visible after clicking should be enough of an indication to a user using a mouse where the focus is? :thinking: What do you think though?

[Screencast from 2024-11-13 15:42:26.webm](https://github.com/user-attachments/assets/ca1cb74d-5f28-444e-995d-134e53cb902a)